docker/docker-compose.ci.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: trufflesuite/ganache-cli:v6.9.1
     expose:
       - 8545
-    command: ['-h=0.0.0.0', '-m=hello unlock save the web', '-i=1984']
+    command: ['-h=0.0.0.0', '-m=hello unlock save the web', '-i=1984', '-e 1000000']
```

Comment:
What is this change for?

Reply:
The test funds 40 ETH into the uniswap pool. The default account balance is only 100. This increases the default account balance so we don't run out of ETH.